.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["mrmlnc.vscode-json5"]
```

Comment:
quick question: should we commit VScode setting files or keep them local?

Reply:
this is recommended extension and intended to be committed so others know they should install this.